CHANGES.md

```diff
@@ -8,6 +8,8 @@ Improvements 🙌:
  - Open an existing DM instead of creating a new one (#2319)
 
 Bugfix 🐛:
+ - Fix issue when updating the avatar of a room
```

Comment:
Please remove this line (it is the same than line 13 which is the latest version)